sr_unix/make_mode.c

```diff
@@ -133,69 +82,9 @@ rhdtyp *make_mode (int mode_index)
 	base_address->lnrtab_len = CODE_LINES;
 	base_address->labtab_len = 1;
 	code = (CODEBUF_TYPE *)base_address->ptext_adr;	/* start of executable code */
-#ifdef __ia64
-	if (dyn_modes_type[mode_index][0] == 'C')
-	{
-		GEN_CALL_C(CODE_ADDRESS(dmode->func_ptr1))		/* line 0,1 */
-	} else
-	{
-		GEN_CALL_ASM(CODE_ADDRESS(dmode->func_ptr1))		/* line 0,1 */
-	}
-#else
 	GEN_CALL(dmode->func_ptr1);			/* line 0,1 */
-#endif /* __ia64 */
-
-#ifdef _AIX
```

Comment:
2. in the commit message below needs to be edited : ia64 -> ia64,AIX,etc.
    - sr_unix/make_mode.c
      1. Removed all capability to create GTM$CI as this "program" is no longer needed.
      2. Removed all #ifdef of ia64.